apps/evm/test/evm/interface/block_interface_test.exs

```diff
@@ -0,0 +1,34 @@
+defmodule EVM.BlockHeaderInfoTest do
```

Comment:
How would you feel about renaming this file to have behavior instead of interface in the name?

Reply:
Oh no! I missed the name of the file? I thought I double checked that. I'll definitely change that

Reply:
Updated in 95df82539e1ea1c1c3706123d5f1359aafdebea3